code/game/verbs/suicide.dm

```diff
@@ -9,6 +9,7 @@
 		return
 	if(confirm == "Yes")
 		suiciding = 1
+		log_game("[key_name(src)] committed suicide at [get_area(src)].")

```

Comment:
We can still find out what they were without it adding jon title.  Its not necessary but probably helpful